src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -1638,80 +1581,82 @@ void InitAirLoops(EnergyPlusData &state, bool const FirstHVACIteration) // TRUE
             state.dataZoneEquip->SupplyAirPath(SupAirPath).NumOutletNodes = NumSupAirPathOutNodes;
 
             // transfer data from the local SupNode array to the SupplyAirPath data structure
-            for (SupNodeIndex = 1; SupNodeIndex <= NumAllSupAirPathNodes; ++SupNodeIndex) {
-                if (state.dataSimAirServingZones->SupNodeType(SupNodeIndex) == DataZoneEquipment::AirNodeType::PathInlet ||
-                    state.dataSimAirServingZones->SupNodeType(SupNodeIndex) == DataZoneEquipment::AirNodeType::Intermediate ||
-                    state.dataSimAirServingZones->SupNodeType(SupNodeIndex) == DataZoneEquipment::AirNodeType::Outlet) {
+            for (int SupNodeIndex = 1; SupNodeIndex <= NumAllSupAirPathNodes; ++SupNodeIndex) {
+                if (supNodeType(SupNodeIndex) == DataZoneEquipment::AirNodeType::PathInlet ||
+                    supNodeType(SupNodeIndex) == DataZoneEquipment::AirNodeType::Intermediate ||
+                    supNodeType(SupNodeIndex) == DataZoneEquipment::AirNodeType::Outlet) {
```

Comment:
BITF? (just practicing)

Reply:
:fire: This is the borderline case and I would like to clean up BITF a bit before resuming making this comment, but as I said ... 🔥